### PR TITLE
Uxd 1260 component health tracker metrics

### PR DESCRIPTION
**Pull request by josh-luong**
### Purpose 🚀
Adding script to dynamically retrieve status information for each component within Paprika, which include:
- bug fix count
- chore count
- commit count
- feat count
- line count
- a11y reports
- commit messages (latest one)
- test coverage stats

These will be displayed in the Sonar application. https://github.com/acl-services/sonar/pull/55

### Notes ✏️
Added `storybook-a11y-report` to dynamically retrieve a11y reports from the storybook itself, which utilized Axe a11y violations. Note: some difficulty adding this, since we needed to **locally** run storybook concurrently alongside this storybook scraper.

This script should be scheduled to run every X days.

The report will be saved as a json file under the `status.json`. In theory, this should be loaded from Sonar repo to get the latest information.

TODO: add additional stats such as Screener reports, or any other stats that are helpful to know.
TODO: create a scheduled job to run the `paprika-status.yml` -- there should be one that was used for testing, but the branch should change to master once this gets approved.

This will be need to be reviewed as a starling day task

### Updates 📦
If you have changed a component's source code (not stories, specs, or docs), before merging your branch run `yarn changeset`. This will prompt you to:
- indicate if changes were patch/minor/major for each modified package
- enter a release message

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

**Comment by allison-c** [comment]
Made the overall time faster, but the storybook a11y reporting is taking too long, even after optimization.
Currently generating all data will take 25mins, generating all data without a11y reports just take 5 mins.
Maybe we can separate the a11y report from the task

**Comment by allison-c** [comment]
Paprika status script has been separated into two, one for the counts, test coverage etc, the other one for a11y issues in storybook.
The first one takes about 8 mins to run. The second one takes 20-30 mins.

File: status-a11y.json [hunk 1]
```diff
@@ -0,0 +1 @@
...D":"label","description":"Ensures every form element has a label"}]},"Tabs":{"a11yReport":[]},"Tag":{"a11yReport":[{"A11yID":"color-contrast","description":"Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds"},{"A11yID":"nested-interactive","description":"Nested interactive controls are not announced by screen readers"}]},"Takeover":{"a11yReport":[]},"Textarea":{"a11yReport":[{"A11yID":"label","description":"Ensures every form element has a label"}]},"TimePicker":{"a11yReport":[{"A11yID":"nested-interactive","description":"Nested interactive controls are not announced by screen readers"}]},"Toast":{"a11yReport":[]},"Tokens":{"a11yReport":[]},"Uploader":{"a11yReport":[{"A11yID":"aria-allowed-attr","description":"Ensures ARIA attributes are allowed for an element's role"},{"A11yID":"color-contrast","description":"Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds"},{"A11yID":"listitem","description":"Ensures <li> elements are used semantically"}]},"helpers":{"a11yReport":[]},"seducer":{"a11yReport":[{"A11yID":"button-name","description":"Ensures buttons have discernible text"}]}}
```
Comment (AndreyChernykh) [inline status-a11y.json:1]: Should this file be tracked in git? What is it for?

File: status.json [hunk 2]
```diff
@@ -0,0 +1 @@
...nt":1823,"version":"2.0.3","logs":"commit 8f4e7e4f0d5326841697778d93126f0f47f2f33f\nAuthor: Allison Chen <38733362+allison-c@users.noreply.github.com>\nDate:   Mon Oct 18 14:45:17 2021 -0700\n\n    Uxd 1328 support ts (#1139)\n    \n    * chore(Paprika): UXD-1328 ts basic setup\n    \n    * chore(Paprika): UXD-1328 ts more configuration\n    \n    * chore(Paprika): UXD-1328 ts more configuration\n    \n    * chore(Paprika): UXD-1328 ts more configuration\n    \n    * chore(Paprika): UXD-1328 bump machine\n    \n    * chore(Paprika): UXD-1328 try fix memory issue\n    \n    * chore(Paprika): UXD-1328 bump eslint\n    \n    * chore(Paprika): fix linter\n    \n    * chore(Paprika): UXD-1328 fix prettier error\n    \n    * chore(Paprika): UXD-1328 fix esm\n    \n    * chore(Paprika): UXD-1328 update rule\n    \n    * chore(Paprika): UXD-1328 bring back types generation script\n    \n    * chore(Paprika): UXD-1328 clean up\n    \n    * fix(Breadcrumbs): UXD-1328 fix storybook example\n    \n    * chore(Paprika): UXD-1328 remove unused declare module\n    \n    * chore(Paprika): UXD-1328 fix version requirement in package.json\n    \n    * chore(Paprika): UXD-1328 cleanup dependency\n"}}
```
Comment (AndreyChernykh) [inline status.json:1]: Should this file be tracked in git? What is it for?

**Review by AndreyChernykh — APPROVED** [review]

**Pull request merged**